.eslintrc.json

```diff
@@ -1,3 +1,35 @@
 {
-  "extends": ["mixmax/node/withJest"]
+  // We can"t use `mixmax/typescript` here because it hardcodes Jest support,
+  // which isn"t used here.
```

Comment:
Unsolicited advice: split this into `mixmax/typescript` and `mixmax/typescript/withJest`, the same way you did with `mixmax/node` and `mixmax/node/withJest`.